DOCS-2305: Email corrections (#1306)
MultiSafepay/docs: content/payment-methods/wallet/alipay/refund-alipay.md

>_Outgoing payments will not be processed if your MultiSafepay balance is insufficient._

> In the event of a failed Alipay refund, email our Support Team at <support@multisafepay.com> to assist you.

Reviewer: ```suggestion
If an Alipay refund fails, email the Support Team at <support@multisafepay.com> 
```
